packages/devtools_app/lib/src/eval_on_dart_library.dart

```diff
@@ -442,6 +463,9 @@ class EvalOnDartLibrary {
   Future<T> addRequest<T>(Disposable isAlive, Future<T> request()) async {
     if (isAlive != null && isAlive.disposed) return null;
 
+    if (!oneRequestAtATime) {
```

Comment:
This speeds up the provider page by maybe 10X as there are a bunch of requests that didn't need to be pipelined.